modules/osdk-prepare-supported-images.adoc

```diff
@@ -54,3 +57,50 @@ to use the supported image:
 ----
 registry.redhat.io/openshift4/ose-kube-rbac-proxy:v{product-version}
 ----
+endif::[]
+
+ifdef::ansible[]
+* Update the project root-level Dockerfile to use supported images. Change the default builder image reference from:
++
+[source,terminal,subs="attributes+"]
+----
+FROM quay.io/operator-framework/ansible-operator:{osdk_ver}
+----
++
+to:
++
+[source,terminal,subs="attributes+"]
+----
+FROM registry.redhat.io/openshift4/ose-ansible-operator:v{product-version}
```

Comment:
Note that in the preview build, the tag comes out as "vBranch Build", but after merging to the `enterprise-4.7` branch here, it will correctly expand as "v4.7".